api-catalog-ui/frontend/src/components/Swagger/SwaggerUI.jsx

```diff
@@ -78,19 +79,24 @@ export default class SwaggerUI extends Component {
                 },
             },
         },
-        // wrapComponents: {
-        //     // prettier-ignore
-        //     // eslint-disable-next-line no-shadow, react/no-unstable-nested-components
-        //     operations: (Original, { React }) => props => { // NOSONAR
-        //         const { selectedService, selectedVersion, tiles } = this.props;
-        //         return (
-        //             <div>
-        //                 <InstanceInfo {...props} selectedService={selectedService} selectedVersion={selectedVersion} tiles={tiles} />
-        //                 <Original {...props} />
-        //             </div>
-        //         );
-        //     },
-        // },
+        wrapComponents: {
+            // prettier-ignore
+            // eslint-disable-next-line no-shadow, react/no-unstable-nested-components
+            operations: (Original, { React }) => props => { // NOSONAR
+                const { selectedService, selectedVersion, tiles } = this.props;
+
+                return (
+                    <div>
+                        {isAPIPortal() &&
```

Comment:
I think it should be `!isAPIPortal()`, we want to display this only when API Portal is false, meaning the Zowe UI will be deployed

Reply:
good catch, fixed